packages/plugins/FileService/src/schema-v2.json

```diff
@@ -9,7 +9,7 @@
         },
         "provider": {
             "type": "string",
-            "enum": ["arweave", "ipfs", "swarm"],
+            "enum": ["arweave", "ipfs"],
```

Comment:
remove it from the schema json will let the old file posted by swarm cannot be displayed. @guanbinrui